src/main/java/pl/javastart/bootcamp/domain/user/role/UserRole.java

```diff
@@ -11,12 +11,20 @@ public class UserRole {
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
-    @ManyToOne
+    @ManyToOne(cascade = CascadeType.ALL)
```

Comment:
Usuwając rolę raczej nie chcemy, żeby był razem z nią usuwany użytkownik, a tutaj będzie m.in. taki efekt

Reply:
tak, to prawda. Po prostu to już były desperackie próby, czy zadziała usuwanie roli.